mandatory/4-tax.js

```diff
@@ -24,10 +22,8 @@ function calculateSalesTax(price) {
 
 function addTaxAndFormatCurrency(price) {
   let totalPrice = calculateSalesTax(price);
-  let decimalNumberOfAbove = Number(totalPrice).toFixed(2);
```

Comment:
Hi Yunus, great work, well done.
As a small suggestion: would be to use lowercase words for starting to write new variables.(eg:Number)

Reply:
> As a small suggestion: would be to use lowercase words for starting to write new variables.(eg:Number)

Here, `Number` is not a variable; rather, it's the [Number function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number#description) that takes a non-number parameter and attempts to convert it to an object of type Number.

Note that using the Number function is not strictly needed here, as `totalPrice` is already a number.